### DISCUSSION TITLE
tree cannot be installed due to network fetching error

### DISCUSSION THREAD

**Discussion by bitweedx64** [Homebrew/discussions#2697] [Everyday usage]
<img width="591" alt="CleanShot 2021-12-29 at 10 02 47@2x" src="https://user-images.githubusercontent.com/83109252/147620450-2c68c6e4-54ab-4246-856a-b6d45a871318.png">
This is what happend. Thought that is a registry related issue?

**Comment by gromgit** [comment]
>```
>$ brew install tree
>==> Downloading https://ghcr.io/v2/homebrew/core/tree/manifests/2.0.0_1
>######################################################################## 100.0%
>[...]
>```

It's either a temporary outage, or something wrong with your network connection. Try again?
